widgets/heatmap/heatmap.go

```diff
@@ -76,7 +88,50 @@ func New(opts ...Option) (*HeatMap, error) {
 // Each call to Values overwrites any previously provided values.
 // Provided options override values set when New() was called.
 func (hp *HeatMap) Values(xLabels []string, yLabels []string, values [][]float64, opts ...Option) error {
```

Comment:
Just occurred to me that the coordinates in values are in the order (y, x) while most coordinates in termdash are in order `(x, y)` as inspired by the `image` library: https://godoc.org/image#Point.

Is the `(y, x)` order intentional? Do you have preferences? Or is it because we are treating it as a matrix of `(rows, cols)`? Whichever we choose, can we make sure it is well documented and the variable names correspond to the chosen order? I.e. if it is a matrix, then we can use the words `rows` and `cols` when referring to coordinates instead of generic counter names like `i` and `j`.

Related to another comment I made below.

Reply:
Good catch. I use `(y, x)` because we treat values as a matrix of `(rows, cols)`.